sites.js

```diff
@@ -2,18 +2,22 @@
 
 // protocol://url.domain.ext
 
+// The `hasRingLink` property refers to the RC logo link that can be found on any site
+// If `hasRingLink` is true then the "explore" feature will include that site.
+// Else "explore" will omit the site.
 const sites = [
-	{ url: 'https://www.jskjott.com', title: 'jskjott', author: 'Jonathan Skjøtt' },
-	{ url: 'https://tomlisankie.com', title: 'Thomas Lisankie', author: 'Thomas Lisankie'},
-	{ url: 'https://www.fekadu.com', title: 'mfekadu', author: 'Michael Fekadu' },
-	{ url: 'http://www.charstiles.com', title: 'charstiles', author: 'Char Stiles' },
-	{ url: 'https://blog.wesleyac.com', title: 'wesleyac', author: 'Wesley Aptekar-Cassels' },
-	{ url: 'https://mclare.blog', title: 'mclare', author: 'Maryanne Wachter'},	
-	{ url: 'https://vaibhavsagar.com', title: 'vaibhavsagar', author: 'Vaibhav Sagar' },
-	{ url: 'https://jon.network', title: 'Jon\' Network', author: 'JonPizza' },
-	{ url: 'https://chirag.io', title: 'Chirag Davé', author: 'Chirag Davé' },
-	{ url: 'https://sequential.me', title: 'Winston Smith', author: 'Winston Smith' },
-  { url: 'https://chatha-sphere.github.io/about', title: 'pchatha', author: 'Prayag (Nikku) Chatha' }
+	{ url: 'https://www.jskjott.com', title: 'jskjott', author: 'Jonathan Skjøtt', hasRingLink: true },
```

Comment:
I think it would be nice to keep the information in sites.js to a minimum and to have only data which the people signing up have to fill in. So maybe we could move hasRingLink to a data structure in the index.js file for now?